src/core/types/decimal192.rs

```diff
@@ -270,16 +613,24 @@ impl Decimal {
     /// - Panic if the number of decimal places is not within [0..SCALE(=18)]
     pub fn round_with_mode(
         &self,
-        decimal_places: i32,
```

Comment:
does not make sense to round with a NEGATIVE number of many decimal places :P ... I used `i32` since that is what Scrypto uses... strange!